portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,51 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Kassen's Portfolio</title>
+    <!-- CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/style.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
     <script src="script.js"></script>
   </head>
-  <body>
+  <body id="body">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <nav class="navbar navbar-default">
+        <div class="container-fluid">
+        <div class="navbar-header">
+        <a class="navbar-brand" href="#">Kassen's Portfolio</a>
+        </div>
+        <ul class="nav navbar-nav">
+        <li class="active"><a href="#">Home</a></li>
+        <li><a href="/projects.html">Projects</a></li>
+        <li><a href="/involvement.html">Involvement</a></li>
+        <li><a href="#">Fun Things</a></li>
+        </ul>
+        </div>
+      </nav>
+
+      <h1>Hi, I'm Kassen!</h1>
+      <p id="demo">I'm a born-and-raised Coloradan and rising junior at Duke. Welcome to my portfolio.</p>
+      <img src="/images/profilephoto.jpeg" alt="Profile Photo">
+
+      <br><br>
+      </hr>
```

Comment:
should this be "\<hr>" instead? here and other usages

Reply:
Changed this and once they actually showed up, I didn't like how they looked so I took them out